content/docs/04-clusters/3-cluster-management/12-palette-webctl.md

```diff
@@ -0,0 +1,140 @@
+---
+title: "Kubectl"
+metaTitle: "Web kubectl CLI on Palette"
+metaDescription: "Web kubectl CLI on Palette for cluster access"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Overview
+
+Palette leverages Kubectl through an in-built command line interface for the users to communicate with their workload clusters. This enables our users to deploy applications, inspect and manage cluster resources, and view logs using the Palette terminal without an external terminal. 
+
+# Usage Scenarios
+
+* Simple Cluster Access
+* Cluster access with OIDC Authentication enabled
+* Cluster access with Spectro Proxy  
+* CLI-Based Cluster Access
+
+## Cluster Access
+
+For general scenario, a user can connect to the cluster directly as below:
+
+<br/>
+
+1. Launch a cluster from the `Project Admin` Console.
+
+
+2. Go the the `Cluster Details` page 
+
+
+3. Click the `Connect` button available at the `Kubernetes Config File.` 
+
+
+4. Wait for the terminal to be launched and start communicating to the cluster using the `kubectl` commands.
+
+
+## Cluster Access with OIDC Authentication Enabled
+
+The cluster access with OIDC authentication enables the clients to verify the end user's identity before establishing cluster connectivity. The user needs to establish an OIDC-based authentication to the cluster through an Identity provider of their choice. To establish cluster access with OIDC authentication, follow the steps below:
```

Comment:
We should also mention.. 
To make this work, they need to add **https://console.spectrocloud.com/v1/shelly/callback** in their IDP app as a callback/redirectURI